### ISSUE TITLE
Are test resources included?

### ISSUE THREAD

**Issue by robross0606** [uklance/gradle-dependency-export#9]
Is this supposed to export test resources or just runtime?  I don't see any of my test resources in the export.

**Comment by robross0606** [comment]
I believe this is related to the "configurations" parameter, but it is unclear what you mean in your readme with the default behavior being "buildscript.configurations + project.configurations".  What is the easiest way to include ALL standard gradle configurations?

**Comment by uklance** [comment]
By default all configurations are exported, including the test configurations for the java plugin. So jars like junit and mockito etc should be in the exported folder

**Comment by robross0606** [comment]
This definitely did not happen. I had to explicitly list all the configs for them to export.  The default behavior only appeared to export “compile”.

**Comment by uklance** [comment]
Hmm... That's strange. I'll have to add a test case to investigate

**Comment by uklance** [comment]
I wonder if this is a timing issue. Are you applying the dependency export plugin before the java plugin?

**Comment by robross0606** [comment]
The mavenDependencyExport section is the last thing in the build.gradle file.  This is a very simple single module project.  Nothing complicated or strange about it, other than perhaps the fact that I'm calling the mavenDependencyExport output into a Zip:

```
task bundleDependencyRepo(type: Zip) {
    baseName = rootProject.name + '-repo'
    from mavenDependencyExport
}
```

**Comment by robross0606** [comment]
I am using Gradle 5.3.1 and the latest JDK 8, if that info helps.

**Comment by robross0606** [comment]
I think you were on to something with the "timing" comment.  I had my mavenDependencyExport section after my bundeDependencyRepo task definition.  When I switched that around so the mavenDependencyExport configuration was before the task definition in my build.gradle file, it appears to now be getting all configurations in the output.

**Comment by robross0606** [comment]
I'm totally lost with the differences in how these plugins are deployed and represented on the servers, depending upon whether the syntax is the old or new style.  In the old style, your plugin is actually at:
```
  dependencies {
    classpath "gradle.plugin.com.lazan:gradle-dependency-export:0.5"
  }
```
and not:
```
  dependencies {
    classpath "com.lazan:gradle-dependency-export:0.5"
  }
```
Yet, the "new" style seemed to work with:
```
plugins {
  id "com.lazan.dependency-export" version "0.5"
}
```
So which one is correct?  Is the structure at `gradle.plugin.com.lazan` or just `com.lazan`?

**Comment by uklance** [comment]
See [here](https://plugins.gradle.org/plugin/com.lazan.dependency-export)

```
buildscript {
  repositories {
    maven {
      url "https://plugins.gradle.org/m2/"
    }
  }
  dependencies {
    classpath "gradle.plugin.com.lazan:gradle-dependency-export:0.5"
  }
}

apply plugin: "com.lazan.dependency-export"
```

**Comment by robross0606** [comment]
Yep, that's where I got the correct syntax.  This is different than what you have in your readme which was the first confusing part.  The new style syntax also doesn't like this at all.  I am trying my entire process with the old-style for now.

**Comment by robross0606** [comment]
This did not work either.  The problem is that the JAR files under /gradle/plugins/ in your export do not have an associated POM file in the export.  This breaks their use as a maven repo.

**Comment by robross0606** [comment]
Sorry, accidentally posted to the wrong thread.  👎
